android/app/src/main/java/org/fossasia/openevent/dbutils/RealmDataRepository.java

```diff
@@ -336,6 +336,7 @@ public Session getSession(String title) {
     }
 
     public RealmResults<Session> getBookMarkedSessionsSync() {
+        Realm realm = Realm.getDefaultInstance();
```

Comment:
Closing realm session does not result in updation of the bookmarks in the widget somehow.